### ISSUE TITLE
Regarding Gulp Build

### ISSUE THREAD

**Issue by brijq** [akveo/blur-admin#353]
I would like to enquire if this gulp app is able to static host? My understanding is about when we run the gulp serve there is a .tmp hidden file being temporary created and serve from there as it can be found in devRelease file.

Therefore when we do a gulp build and expect it to work on the static host, it does not make sense which is possible without browser sync and ..etc, how can it be done?

Please advise

**Comment by iba-jsykes** [comment]
After your clean, then build.. take the contents of the /release folder and put them on the server of your choice, NGINX, Apache, etc .  Injection of browser-sync is only done on gulp serve or "like" tasks.

**Comment by brijq** [comment]
Ok noted that which in short it must be dynamically hosted right?

**Comment by iba-jsykes** [comment]
No, very much static.  Your build produces all the css, js, and html files, ( and other assets ) for it to run independently of any kind of build process.   Use the release folder, after the build gulp task.  You put that folder on any static host and your app will load.

**Comment by brijq** [comment]
I understand where you are coming from however the issue is the the release folder is kind of different from the actual folder where you run it dynamically. In a way. The release folder after building seems not completed. For example like some js files. 

I understand the entire process of building actually the key idea here is that it transform js, css into min files however it feels incomplete.

All css html front end files are properly build however it is the js files that are incomplete.

**Comment by elyohan14** [comment]
Any solution for this?
